samples/saga/ravendb-sequence-generator/Version_5/Sample/SequenceSagaData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using NServiceBus.Saga;
+
+#region OrderSagaDataRavenDB
+
+public class SequenceSagaData : IContainSagaData
```

Comment:
yes, I sincerely don't know why most of the saga samples does not do that, I'll update all.